test/k8sT/Services.go

```diff
@@ -264,29 +291,40 @@ var _ = Describe("K8sServicesTest", func() {
 		}
 	}
 
-	waitPodsDs := func() {
-		groups := []string{testDS, testDSClient, testDSK8s2}
-		for _, pod := range groups {
-			err := kubectl.WaitforPods(helpers.DefaultNamespace, fmt.Sprintf("-l %s", pod), helpers.HelperTimeout)
-			ExpectWithOffset(1, err).Should(BeNil())
-		}
-	}
-
 	Context("Checks ClusterIP Connectivity", func() {
+		const (
+			serviceName     = "app1-service"
+			serviceNameIPv6 = "app1-service-ipv6"
+			echoServiceName = "echo"
+			echoPodLabel    = "name=echo"
+			// echoServiceNameIPv6 = "echo-ipv6"
+		)
 
 		var (
-			demoYAML    string
-			echoSVCYAML string
+			demoYAML             string
+			demoYAMLDualStack    string
+			echoSVCYAML          string
+			echoSVCYAMLDualStack string
 		)
 
 		BeforeAll(func() {
 			demoYAML = helpers.ManifestGet(kubectl.BasePath(), "demo.yaml")
 			echoSVCYAML = helpers.ManifestGet(kubectl.BasePath(), "echo-svc.yaml")
 
 			res := kubectl.ApplyDefault(demoYAML)
-			res.ExpectSuccess("unable to apply %s", demoYAML)
+			Expect(res).Should(helpers.CMDSuccess(), "unable to apply %s", demoYAML)
 			res = kubectl.ApplyDefault(echoSVCYAML)
-			res.ExpectSuccess("unable to apply %s", echoSVCYAML)
+			Expect(res).Should(helpers.CMDSuccess(), "unable to apply %s", echoSVCYAML)
+			if dualStackSupportEnabled {
```

Comment:
This looks odd, we can't "know" the order the variable will be initialized so it would be better executing `helpers.DualStackSupported()` directly here.